New Early Years and Further Education pages (#364)
DFE-Digital/get-into-teaching-content: content/further-education-teacher-training.md

---
title: "Further education teacher training"
image: false
backlink: "../"
keywords:
  - FE
  - Further Education
  - FE Advice
---

## Become a further education teacher

Being a further education (FE) teacher means you’ll teach students aged 14 and above. Further education covers a hugely diverse range of subjects and qualifications.

> As an FE teacher, you’ll be improving your students’ qualifications and future job prospects - as well as inspiring those who simply want to enjoy building new skills and knowledge.

Reviewer: ```suggestion
As an FE teacher, you’ll improve your students’ qualifications and job prospects. You'll also inspire people who want to enjoy building new skills and knowledge.
```

FE qualifications range from basic literacy and numeracy to higher-level apprenticeships and technical qualifications, including the <a href='https://www.gov.uk/government/publications/introduction-of-t-levels/introduction-of-t-levels'>new T levels</a>, which are vocational alternatives to A levels.

FE students come from a range of backgrounds and levels of experience. People come to FE from local schools, the community, industry and local businesses.

> Because there’s no national curriculum for FE, courses are often tailored to meet employer needs and are delivered in a range of different settings, such as colleges, community centres, private training centres or workplaces.

Reviewer: ```suggestion
There’s no national curriculum for FE, so courses are often tailored to meet employer needs. They're delivered in a range of different settings, such as colleges, community centres, private training centres or workplaces.
```

### How to become a further education teacher

> Because the courses and qualifications are so varied, the qualifications teachers need are too. There are qualifications on education and training, as well as the specialised subject-area skills or qualifications that a teacher is expected to have. You might even be able to gain new qualifications while you teach.

Reviewer: ```suggestion
The qualifications you need to become an FE teacher depend on the type of course you'd like to teach. For example, you can get qualifications in education and training, as well as qualifications in specific subjects. 

You can even gain new qualifications once you start teaching. 
```

> For more information, visit the <a href='https://www.feadvice.org.uk/i-want-work-fe-skills-sector'>FE Advice website</a> or phone 0300 303 1877.

Reviewer: ```suggestion
Visit <a href='https://www.feadvice.org.uk/i-want-work-fe-skills-sector'>Further Education Advice</a> or ring their support service on 0300 303 1877.
```
